### PR TITLE
First test script to install sysl using homebrew #minor

### PR DESCRIPTION
**Pull request by ashwinsajiv**
Adding a test script to the sysl repo to install sysl using homebrew. Does not impact any other file.

### CODE REVIEW

**Comment by cuminandpaprika** [comment]
Got a question, what's the reason that the homebrew script needs to live in the same repo as Sysl? 

Okay, post our discussion, sounds like the ruby script is building and copying the file to the gopath, which we're currently not doing in the makefile.

**Comment by ashwinsajiv** [comment]
> Got a question, what's the reason that the homebrew script needs to live in the same repo as Sysl?
> 
> Okay, post our discussion, sounds like the ruby script is building and copying the file to the gopath, which we're currently not doing in the makefile.

Yes, also the make file install has access to sysl binary already through the dist folder whereas the homebrew installation has the zip file of the release which does not have the binary. Hence, the decision to have a seperate script to do this. The script is also intended to do more - such as -  install go and create gopath if it doesn't exist already.

File: test.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    end
+    dest = nil
+  end
+end
+
+Src = File.join(ENV['HOME'], 'Library', 'Caches', 'Homebrew', 'sysl-0.6.3', 'cmd', 'sysl')
+Dir.chdir Src
+system("go", "build")
+Dest = File.join(ENV['HOME'],'go', 'bin')
+FileUtils.cp(File.join(Src, 'sysl') ,Dest)
```
Comment (camh-anz) [inline test.rb]: Missing eol at eof

Comment (ashwinsajiv) [reply test.rb]: I have an issue raised which keeps track to rename the file and to clean things up.

**Pull request merged**